### PR TITLE
Demo server guide

### PR DESCRIPTION
**Pull request by CarlosNZ**
Just quickly added this doc for updating the demo server.

### CODE REVIEW

File: docs/internal/demo-server-guide.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Upload to Docker hub
+
+- Login to docker hub (web site) to see the images in there:
+  - msupplyfoundation - password in Bitwarden (Shared-Admin)
+- List local images to get the “image ID” and "tag name"  
+  `docker image ls`
+- Create tag:  
+  `docker tag <image_id> /<docker-repo>:<tag-name>`
+- Push new tag:  
+   `docker push <image_id> /<docker-repo>:<tag-name>`
```
Comment (nmadruga) [inline docs/internal/demo-server-guide.md:32]:
```suggestion
  `docker tag <image_id> <organisation>/<docker-repo>:<tag-name>`
- Push new tag:  
   `docker push <image_id> <organisation>/<docker-repo>:<tag-name>`  
```

File: docs/internal/demo-server-guide.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## To restart an instance
+
+- Run `sudo docker stop <name>` (name from above, or can use container id)
+- Remove container: `sudo docker rm <name>`
+- Re-run as above. Note: this resets the container to initial state, including database reset. If you want to preserve existing data, you’ll need to take a snapshot first, then reload after restart.
+
+### To document:
+
+- How do I check logs?
+- How to check Env vars?
```
Comment (nmadruga) [inline docs/internal/demo-server-guide.md:77]: $ printenv

Right?

**Review by nmadruga — CHANGES_REQUESTED** [review]
Nice to have that in here. Just one missing item (probably from my original document)

File: docs/internal/demo-server-guide.md (hunk 2 above)
Comment (CarlosNZ) [reply docs/internal/demo-server-guide.md:77]: Great, thanks!!

**Review by nmadruga — APPROVED** [review]

**Comment by CarlosNZ** [comment]
FYI -- I just realised that this should really be in back-end docs, so I've moved it across to there (done directly in `develop`)

**Pull request merged**